### DISCUSSION TITLE
What exactly is `ibbBtn`?

### DISCUSSION THREAD

**Discussion by just-a-hriday** [ScratchAddons/ScratchAddons#3785] [Help]
I've been studying the code for [`material-forum-editor-buttons`](https://github.com/ScratchAddons/ScratchAddons/blob/dcb8d88e865c1067ad354b549a5670935171c627/addons/material-forum-editor-buttons/icons.css), and I noticed [this line](https://github.com/ScratchAddons/ScratchAddons/blob/dcb8d88e865c1067ad354b549a5670935171c627/addons/material-forum-editor-buttons/icons.css#L71). What is `ibbBtn`? It seems to have something to do with the upload button, but I can't find anything about it anywhere.

**Comment by mxmou** [comment]
The selector has been there since the userstyle [was made](https://github.com/easrng/material-scratch-forum-editor-buttons/commit/a6f1750bd9414ee6e4a8bc55e9b1f6255c8363e0), which means it isn't related to the image upload button.

**Reply by CST1229** [reply]
It might have been for compatibility with a separate userscript that is not part of SA.

**Reply by just-a-hriday** [reply]
> The selector has been there since the userstyle [was made](https://github.com/easrng/material-scratch-forum-editor-buttons/commit/a6f1750bd9414ee6e4a8bc55e9b1f6255c8363e0), which means it isn't related to the image upload button.

But then why is it there? I don't think it would have been for compatibility with something outside SA. But are there any addons that have anything to do with it?

**Reply by mxmou** [reply]
SA didn't exist in May 2020.
